src/templates/common-annotations.yaml

```diff
@@ -4,4 +4,5 @@
     sourceGenerationRulePath: {{generation_rule_file_path}}
     config.runwhen.com/last-updated-by: workspace-builder
```

Comment:
is this in use at all then? We need to see if `internal.runwhen.com/generated-by: "{{generated_by}}"` supersedes it or if it is used at all by PAPI to help with the last known editor. I think this was originally put in place with that idea in mind, but I don't think the PAPI side was ever touched.